### DISCUSSION TITLE
Images won't display

### DISCUSSION THREAD

**Discussion by middlepath79** [community/community#22694] [Other Feature Feedback, Questions, & Ideas]
<p>Images used for my website won’t display after deployment but it shows locally any tips please.<br>
I wonder what the problem is help</p>
<p><a href="http://Middlepath79.github.io" class="onebox" target="_blank" rel="nofollow noopener">Middlepath79.github.io</a></p>

**Comment by SethClydesdale** [comment, marked as answer]
<p>Hello <a class="mention" href="/middlepath79">@middlepath79</a>,</p>
<p>In the <strong>src</strong> attribute for your images, you’re referencing the <strong>images</strong> folder, however, there is no such folder in your respository. It looks like your images are in the root of your repo instead. You can solve the images not displaying in two ways:</p>
<ol>
<li>create a folder named “images” and add all of your image files to it.</li>
<li>or delete “images/” from your <code>&lt;img&gt;</code> src attributes.</li>
</ol>

**Comment by middlepath79** [comment]
<p>Resolved thanks once more …:clap:</p>
